quiz/models.py

```diff
@@ -16,11 +14,10 @@ def __str__(self):
 
 
 class Results(models.Model):
-    user = models.ForeignKey(User, on_delete=CASCADE, null=False)
+    user = models.ForeignKey(CustomUser, on_delete=CASCADE, null=False)
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import